autoscale-core/src/main/java/com/hpe/caf/autoscale/core/EmailDispatcher.java

```
@@ -0,0 +1,114 @@
+                    props.put("mail.smtp.port", port);
+                    props.put("mail.smtp.starttls.enable", "true");
+                    props.put("mail.smtp.auth", "true");
+
+                    // Create a Session object to represent a mail session with the specified properties. 
+                    final Session session = Session.getDefaultInstance(props);
+
+                    // Create a message with the specified information. 
+                    final MimeMessage msg = new MimeMessage(session);
+                    msg.setFrom(new InternetAddress(emailAddressFrom, FROMNAME));
+                    msg.setRecipient(Message.RecipientType.TO, new InternetAddress(emailAddressTo));
+                    msg.setSubject(SUBJECT);
+                    msg.setContent(messageContent, "text/html");
+
+                    // Create a transport.
+                    final Transport transport = session.getTransport();
+
+                    // Send the message.
+                    try {
+                        LOG.debug("Sending email...");
+
+                        // Connect to the SMTP server using username and password specified above.
+                        transport.connect(host, username, password);
+
+                        // Send the email.
+                        transport.sendMessage(msg, msg.getAllRecipients());
+                        lastSent = date;
+                        lastPriorityThreshold = priorityThreshold;
+                        LOG.debug("Email sent!");
+                    } catch (final MessagingException ex) {
+                        LOG.error("The email was not sent.", ex);
+                    } finally {
+                        // Close and terminate the connection.
+                        transport.close();
+                    }
+                } catch (final Exception ex) {
+                    LOG.error("The email was not sent.", ex);
+                }
```

This function is called `dispatchEmail()`, not `tryToDispatchEmail()`.  If it fails it should throw an exception so that the caller can decide what to do (which may well be to log the failure and move on - but only at that level can the decision be made - this component should assume that there may be a backup alert system).